CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased] - yyyy-mm-dd
+
+
+
+
+[1.0.0]: https://github.com/digipolisantwerp/authz_module_nodejs/tree/v1.0.0
```

Comment:
Voor mij ok, maar misschien kan je deze link mee onderaan zetten onder `[Unreleased]`?

Reply:
ik heb de unreleased link bij de unreleased block gezet. Dan is het consistent.
Ik vind het persoonlijk gemakkelijker dat de info per release gegroepeerd staat in de file.